packages/react-components/react-persona/README.md

```diff
@@ -2,10 +2,6 @@
 
 **React Persona components for [Fluent UI React](https://react.fluentui.dev/)**
 
-## STATUS: WIP 🚧
-
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
-
```

Comment:
This one should not be removed, `react-persona` is still unstable.

Reply:
reverted